ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -124,6 +122,31 @@ def provider_and_native_client_installation(
         wait_for_machineconfigpool_status(node_type="all")
         log.info("All the nodes are upgraded")
 
+        # Mark master nodes schedulable if mark_masters_schedulable: True
+        if config.ENV_DATA.get("mark_masters_schedulable", False):
+            path = "/spec/mastersSchedulable"
+            params = f"""[{{"op": "replace", "path": "{path}", "value": true}}]"""
+            self.scheduler_obj.patch(params=params, format_type="json"), (
+                "Failed to run patch command to update control nodes as scheduleable"
+            )
+            # Allow ODF to be deployed on all nodes
+            log.info("labeling all nodes as storage nodes")
+            label_nodes(nodes=node_objs, label=constants.OPERATOR_NODE_LABEL)
+            worker_node_objs = get_nodes(node_type=constants.WORKER_MACHINE)
+            no_of_worker_nodes = len(worker_node_objs)
+        else:
+            log.info("labeling worker nodes as storage nodes")
+            label_nodes(nodes=worker_node_objs, label=constants.OPERATOR_NODE_LABEL)
+
+        disks_available_on_worker_nodes_for_cleanup = disks_available_to_cleanup(
+            worker_node_objs[0]
+        )
```

Comment:
Shouldn't we go over all the worker nodes(and not just one) to take all the available disks to clean up?

Reply:
I have executed this for one worker node because I am setting that value as ["spec"]["storageDeviceSets"][0]["count"] is storage_cluster yaml. 
Have seen that in general the disks count are same for all the worker nodes, so  thought I can collect this value from any one.